CMakeLists.txt

```diff
@@ -184,11 +119,6 @@ if(UNIX)
   set(CPACK_DEBIAN_PACKAGE_MAINTAINER "Microsoft - Azure Storage")
   set(CPACK_DEBIAN_PACKAGE_DEPENDS "fuse")
   set(CPACK_DEBIAN_PACKAGE_DESCRIPTION "blobfuse 1.2.4 - FUSE adapter for Azure Blob Storage")
```

Comment:
this should be the same version as above

Reply:
Addressed. Will push with other review changes.

Reply:
Will change version to 1.2.5 as suggested earlier in other variables too.